js/block.js

```diff
@@ -3674,6 +3674,23 @@ class Block {
                     '" />';
                 labelElem.classList.add("hasKeyboard");
                 this.label = docById("numberLabel");
+                this.label.addEventListener("input", function () {
```

Comment:
Maybe I am missing something, but is this somehow restricted to graphics blocks or it is an across the board limit on number block size?

Reply:
Why not simply check to ensure the value >= -10000 and =< 10001 in one if statement?

Reply:
> Maybe I am missing something, but is this somehow restricted to graphics blocks or it is an across the board limit on number block size?

@walterbender, upon another look it seems like this problem is not limited to the forward block only, if a number bigger than or equal to 100000 is put in the number block the browser starts to hang. Should i just add the warning whenever the number block is used in block.js and then add a check in the number block class ?